### ISSUE TITLE
Slider + AccelManager conflict 

### ISSUE THREAD

**Issue by PierreHei** [wanadev/PhotonUI#51]
The slider element blocks all keyboard inputs if it's focused, even with the safe argument passed to false in the addAccel method, like in the following code :

```javascript
var slider = new photonui.Slider({
    fieldVisible: true,
    min: 0, max: 100,
    step: 10,
    value: 50
});
photonui.domInsert(slider, "demo");

var label = new photonui.Label({text: ""});
photonui.domInsert(label, "demo");

var accel = new photonui.AccelManager();

accel.addAccel("accel3", "ctrl + q", function() {
    label.text += "'Ctrl + Q' accelerator\n";
}, false);
```

**Comment by Breush** [comment]
So it's closed by f6a1751, right?

**Comment by flozz** [comment]
Yes `f6a1751` fixes this issue. This issue will automatically be closed when the fix will be merged into `master` (for the next release).
